gprofiler/profilers/java.py

```diff
@@ -312,10 +343,7 @@ def __init__(
         self.java_collect_jvm_flags = java_collect_jvm_flags
 
     def make_application_metadata(self, process: Process) -> Dict[str, Any]:
-        if is_java_basename(process):
-            version = get_java_version(process, self._stop_event)
-        else:
-            version = "not /java"
+        version = get_java_version(process, self._stop_event) or "not /java"
```

Comment:
Is "not /java" still correct? We do handle "not /java" now. I'd use "/java not found"?